rss_parser/_parser.py

```diff
@@ -1,43 +1,44 @@
-from typing import ClassVar, Optional, Type
+from typing import Optional, Type
 
 from xmltodict import parse
 
 from rss_parser.models import XMLBaseModel
+from rss_parser.models.atom import Atom
 from rss_parser.models.rss import RSS
 
 # >>> FUTURE
 # TODO: May be support generator based approach for big rss feeds
 # TODO: Add cli to parse to json
 # TODO: Possibly bundle as deb/rpm/exe
-# TODO: Atom support
-# TODO: Older RSS versions?
+# TODO: Older Atom versions
+# TODO: Older RSS versions
 
 
 class Parser:
-    """Parser for rss files."""
-
-    schema: ClassVar[Type[XMLBaseModel]] = RSS
```

Comment:
@dhvcc Making the changes as you mentioned. I am curious why to use a the custom decorator  `abstract_class_attributes` instead of setting a metaclass and abstract method instead?

For example:
```
from abc import ABCMeta, abstractmethod
...

class BaseParser(metaclass=ABCMeta):
    ...
    @property
    @classmethod
    @abstractmethod
    def schema(cls) -> type[XMLBaseModel]:
        return XMLBaseModel
```

Reply:
@ddkasa no real preference, I'd use metaclasses, but I just found this snipped with a decorator and wanted to test if it works